_topic_maps/_topic_map_ms.yml

```
@@ -422,24 +422,24 @@ Name: Running applications
 Dir: microshift_running_apps
 Distros: microshift
 Topics:
-- Name: Embedding MicroShift containers for offline deployments
-  File: microshift-embed-microshift-offline-deploy
+#- Name: Embedding MicroShift containers for offline deployments
+#  File: microshift-embed-microshift-offline-deploy
+- Name: Using kustomize to deploy applications
```

There's a little inconsistency across the content with "Kustomize" vs "kustomize" (excluding "`kustomize`", the tool name). I think you want a capital "K" anytime it's not in backticks.
```suggestion
- Name: Using Kustomize to deploy applications
```